notebook/static/notebook/js/codecell.js

```diff
@@ -509,10 +509,10 @@ define([
     };
 
 
-    CodeCell.prototype.clear_output = function (wait) {
-        this.output_area.clear_output(wait);
-        this.set_input_prompt();
+    CodeCell.prototype.clear_output = function (wait, ignore_queue) {
```

Comment:
Should `ignore_queue` have a default argument since prior calls to this would not have had an argument? Perhaps `undefined` is ok to pass through to `this.output_area.clear_output`?